config/config_test.go

```diff
@@ -116,12 +121,18 @@ backend:
 compression:
   type: "snappy"
 metrics:
-  type: "none"
   influx:
     host: "default-metrics-host"
     database: "default-metrics-database"
     username: "metrics-username"
     password: "metrics-password"
+    enabled: true
+  prometheus:
+    port: 8080
+    namespace: "default-prometheus-namespace"
+    subsystem: "default-prometheus-subsystem"
+    timeout_ms: 100
+    enabled: true
```

Comment:
Please update config.yaml this way too, except to set the enabled flags to false.